package.json

```diff
@@ -135,23 +144,22 @@
     "@elastic/charts": "^30.2.0",
     "@elastic/datemath": "^5.0.3",
     "@elastic/eslint-config-kibana": "^0.15.0",
-    "@svgr/core": "^7.0.0",
-    "@svgr/plugin-svgo": "^7.0.0",
-    "@svgr/plugin-jsx": "^7.0.0",
-    "@types/classnames": "^2.2.10",
-    "@types/enzyme": "^3.10.5",
+    "@svgr/core": "^8.0.0",
+    "@svgr/plugin-svgo": "^8.0.1",
+    "@svgr/plugin-jsx": "^8.0.1",
```

Comment:
For reviewers, although the upgrade was to a major version, the current functionality of 
`compile-icons` is not affected